Model/Paypal_Payment_Method.php

```diff
@@ -48,21 +33,21 @@ public function __construct()
      */
     public function getConfig(): Config
     {
-        $payment = oxNew(\OxidEsales\Eshop\Application\Model\Payment::class);
-        $payment->load(self::NAME);
-        $config = new Config(
-            $payment->oxpayments__wdoxidee_apiurl->value,
-            $payment->oxpayments__wdoxidee_httpuser->value,
-            $payment->oxpayments__wdoxidee_httppass->value
+        $oPayment = oxNew(Payment::class);
+        $oPayment->load(self::getName(true));
+        $oConfig = new Config(
+            $oPayment->oxpayments__wdoxidee_apiurl->value,
+            $oPayment->oxpayments__wdoxidee_httpuser->value,
+            $oPayment->oxpayments__wdoxidee_httppass->value
         );
         $oPaymentMethodConfig = new PaymentMethodConfig(
-            PayPalTransaction::NAME,
-            $payment->oxpayments__wdoxidee_maid->value,
-            $payment->oxpayments__wdoxidee_secret->value
+            self::getName(),
```

Comment:
we can keep the `PayPalTransaction::NAME` we know that it is this one. performance wise better than calling a function which does string manipulation and more obvious what the value is.

Reply:
I think it is more obvious to use the function for both values in line 37 and 44